\hypertarget{classconans_1_1errors_1_1NotFoundException}{}\doxysection{conans.\+errors.\+Not\+Found\+Exception Class Reference}
\label{classconans_1_1errors_1_1NotFoundException}\index{conans.errors.NotFoundException@{conans.errors.NotFoundException}}


Inheritance diagram for conans.\+errors.\+Not\+Found\+Exception\+:
% FIG 0


Collaboration diagram for conans.\+errors.\+Not\+Found\+Exception\+:
% FIG 1
\doxysubsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
\mbox{\Hypertarget{classconans_1_1errors_1_1NotFoundException_a5139524d0fabdcbba11cd1b188db0d23}\label{classconans_1_1errors_1_1NotFoundException_a5139524d0fabdcbba11cd1b188db0d23}} 
def {\bfseries \+\_\+\+\_\+init\+\_\+\+\_\+} (self, $\ast$args, $\ast$$\ast$kwargs)
\end{DoxyCompactItemize}
\doxysubsection*{Public Attributes}
\begin{DoxyCompactItemize}
\item 
\mbox{\Hypertarget{classconans_1_1errors_1_1NotFoundException_aeb90f8ca1b6f3bea605d7f1048f6ab01}\label{classconans_1_1errors_1_1NotFoundException_aeb90f8ca1b6f3bea605d7f1048f6ab01}} 
{\bfseries remote}
\end{DoxyCompactItemize}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}    404 error
\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/lib/python3.\+10/site-\/packages/conans/errors.\+py\end{DoxyCompactItemize}
